sensors/omron_d6t/omron_d6t.py

```diff
@@ -0,0 +1,96 @@
+import time, crcmod.predefined
```

Comment:
I was taught to choose between `from _ import _` and `import _` style importing depending on if what is being imported makes sense on its own or if it needs context. By that logic I would import `sleep` directly but it is generally understood.